bluemira/builders/EUDEMO/reactor.py

```diff
@@ -260,7 +268,7 @@ def build_in_vessel_component_shapes(self, component_tree: Component):
         bluemira_print(f"Starting design stage: {name}")
 
         default_variables_map = {
-            "x1": {"value": "r_fw_ib_in"},  # ib radius
+            "x1": {"value": "r_fw_ib_in", "fixed": True},  # ib radius
```

Comment:
We want to pin this, and probably also the outer radius. There are some robustness issues at the moment